wep-template.rst

```diff
@@ -0,0 +1,46 @@
+
+.. Template instructions:
+
+.. 0 Copy this file and rename it according to WEP filename convention: [WEP####-short-descriptive-name.rst]
+.. 1. Replace all values in square brackets [] with actual values.
+.. 2. Remove optional fields that you do not need. 
+.. 3. Follow reStructured Text markup conventions: http://docutils.sourceforge.net/rst.html
+.. 4. Remove these instructions before submitting the PR.
+.. 5. Submit the WEP as a PR and make sure it follows the guidelines of WEP0000: [URL]
+
+.. For questions and help in creating the WEP, email support@writethedocs.org
+
+:WEP-Number: [####]
+:Created: [YYYY-MM-DD]
+:Last-Modified: [YYYY-MM-DD]
+:Author:  - [FULL NAME 1]
+          - [FULL NAME 2]
+          - ...
+          - ...
+:Status: Draft 
+:Replaces: [OPTIONAL]
+:Superceded-By: [OPTIONAL]
+
+WEP[####] - [Short but descriptive title]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Abstract
+========
+
+[Short summary of the WEP purpose. 1-2 paragraphs maximum. This is the TL;DR.]
```

Comment:
```suggestion
[Short summary of the WEP purpose. 100 words maximum. This is the TL;DR.]
```

I'm suggesting something more concrete numbers here. It doesn't have to be these numbers specifically but I think having concrete length requirements is a kindness to authors and editors—or, in this case, the community.

Reply:
Works for me.